### PR TITLE
feat: add sticky footer to bottom of all pages

### PR DESCRIPTION
**Pull request by echl**
image

### CODE REVIEW

File: packages/client/hmi-client/src/App.vue [hunk 1]
```diff
@@ -118,4 +121,17 @@ main {
 .data-explorer {
 	z-index: 3;
 }
+
+footer {
+	width: 100%;
+	height: 48px;
+	background-color: #344054;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/App.vue:128]: Could you use a CSS property for it? One of the `--grey-X0`

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/client/hmi-client/src/App.vue (hunk 1 above)
Comment (echl) [reply packages/client/hmi-client/src/App.vue:128]: we don't have an existing property but I will create one

Comment (echl) [reply packages/client/hmi-client/src/App.vue:128]: or do you just want to use an existing colour? I grabbed the colour directly from figma

Comment (YohannParis) [reply packages/client/hmi-client/src/App.vue]: `--gray-800` should be the closest one.

**Review by YohannParis — APPROVED** [review]

**Pull request merged**